packages/react-scripts/scripts/utils/forkSsr.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * This script runs a Webpack compiler to watch the SSR build. It is designed to
+ * be run by child_process.fork from the start-ssr script.
+ */
+
+const fs = require('fs');
+const webpack = require('webpack');
+const { prepareUrls } = require('react-dev-utils/WebpackDevServerUtils');
+const paths = require('../../config/paths');
+
+const ssrConfigFactory = require('../../config/webpack.config.ssr');
+const statusFile = require('./statusFile');
+const { DebugCompilerUi, ProcessSendCompilerUi } = require('./MultiCompilerUi');
+const { createCustomCompiler } = require('./customWebpackUtils');
+
+const useYarn = fs.existsSync(paths.yarnLockFile);
+
+const config = ssrConfigFactory('development');
+const appName = require(paths.appPackageJson).name;
+
+const useTypeScript = fs.existsSync(paths.appTsConfig);
+const tscCompileOnError = process.env.TSC_COMPILE_ON_ERROR === 'true';
+
+// We don't start a DevServer so we can use dummy URLs
+const urls = prepareUrls(
+  'http', // protocol
+  'localhost', // HOST
+  '3000', // port
+  paths.publicUrlOrPath.slice(0, -1)
+);
+
+// We don't start a DevServer so we can drop devSocket messages
+const devSocket = {
```

Comment:
you can just make `devSocket` be `undefined` and not have to populate this with empty objects. 

See: https://github.com/Skyscanner/backpack-react-scripts/pull/98/files#diff-134cccbf028a8c082027fe3dbaa6ec8ab507f6771d8f6f9eb8d2be049a7e5462L68